File: mgwt/src/main/java/com/propertycross/mgwt/locations/Location.java
package com.propertycross.mgwt.locations;

public final class Location {

    private final String displayName;
    private final String name;

    public Location(String displayName, String name)
    {
        this.displayName = displayName;
        this.name = name;
    }

    public String getDisplayName()
    {
        return displayName;
    }

    public String getName()
    {
        return name;
    }

    @Override
    public String toString()
    {
        return "Location{" + "longTitle=" + displayName + ", name=" + name + '}';
    }

}

File: native/ios/PropertyCross/Model/Location.m
//
//  Location.m
//  PropertyCross
//
//  Created by Colin Eberhardt on 17/10/2012.
//  Copyright (c) 2012 Colin Eberhardt. All rights reserved.
//

#import "Location.h"
#import "NSDictionary+DictionaryJSONHelpers.h"

@implementation Location

+ (id)locationFromJSONData:(NSDictionary *)jsonData
{
    Location* location = [[Location alloc] init];
    
    location.displayName = [jsonData NSStringForKey:@"long_title"];
    location.name = [jsonData NSStringForKey:@"place_name"];
    
    return location;
}

@end
